fstest/fstests/gen_tests.go

```diff
@@ -164,5 +164,6 @@ func main() {
 	generateTestProgram(t, fns, "Box")
 	generateTestProgram(t, fns, "QingStor", buildConstraint("!plan9"))
 	generateTestProgram(t, fns, "AzureBlob", buildConstraint("go1.7"))
+	generateTestProgram(t, fns, "Cache", "")
```

Comment:
You need to lose the `, ""` here - I change the interface recently

Reply:
done